### DISCUSSION TITLE
クエスト追加案

### DISCUSSION THREAD

**Discussion by RL-Nadir** [sikabane-works/bakabakaband#1660] [Ideas]
完全にAAネタにすぎず、さすがにissueにするのは気が引けたのでこちらに不法投棄いたします。
クエストにするもよし、レアな汚いvaultに転用するもよしということで。
![image](https://user-images.githubusercontent.com/79453028/147867888-059d9328-1cf3-4068-9ce4-ba1f82f3bb6f.png)
[q0clone0.txt](https://github.com/sikabane-works/bakabakaband/files/7798659/q0clone0.txt)

**Comment by deskull-m** [comment]
やりますねえ！（賞賛）クエスト追加体制整ったら前のと一緒にぶち込むのでまって……

**Reply by RL-Nadir** [reply]
こちらもv_infoにしました
（…が新地形が入っている影響なのか、手元のalpha30での出現は確認できて）ないです
[v_Big&MidYajue.txt](https://github.com/sikabane-works/bakabakaband/files/8426501/v_Big.MidYajue.txt)

**Reply by deskull-m** [reply]
毎度アリガトナス。そうすなこれも固定部屋にしてみます。

**Comment by RL-Nadir** [comment]
下北沢ダンジョンに出やすいvault案ということで…
（とりあえず絵を出すためにq_infoをいじった）
  
![image](https://user-images.githubusercontent.com/79453028/148682056-9bf2b1d9-eb96-4afd-8dfc-5f97eaaa0588.png)
[q0000001.txt](https://github.com/sikabane-works/bakabakaband/files/7835015/q0000001.txt)

**Reply by deskull-m** [reply]
ドシドシアリガトナス！

**Comment by RL-Nadir** [comment]
例によって不法投棄します
よろしくナス
余計なのは取っ払って、中央に輝きくんを置いてやったらいいんじゃないでしょうか
![image](https://user-images.githubusercontent.com/79453028/159919986-e37964d8-7be2-49de-be2b-4abbe8c77c0a.png)
[q0000001.txt](https://github.com/sikabane-works/bakabakaband/files/8341535/q0000001.txt)

**Reply by deskull-m** [reply]
ありがとなす！全然クエスト追加できんでセンセンシャル！

**Reply by RL-Nadir** [reply]
クエストにはならんだろうと思いますので、v_infoにしておきました。

[Brilliant_Life_vault.txt](https://github.com/sikabane-works/bakabakaband/files/8350379/Brilliant_Life_vault.txt)
